qoffer-infrastructure/src/main/groovy/life/qbic/portal/portlet/qOfferPortlet.groovy

```diff
@@ -12,8 +12,10 @@ import groovy.transform.CompileStatic
 import groovy.util.logging.Log4j2
 
 /**
- * Entry point for portlet qOffer_2.0. This class derives from {@link QBiCPortletUI}, which is found in the {@code portal-utils-lib} library.
+ * Entry point for portlet qoffer_2.0. This class derives from {@link QBiCPortletUI}, which is found in the {@code portal-utils-lib} library.
```

Comment:
Nit:
```suggestion
 * Entry point for the application. This class derives from {@link QBiCPortletUI}.
```
Less prone to changes due to renaming etc. and still specific enough. Also the location of the `QBiCPortletUI` will automatically be linked in the javadoc and can be viewed by Eclipse, IntelliJ and other IDEs directly.

Reply:
Good Idea, the basic description was autogenerated by qube and i only changed the portlet name.